src/coprocessor/dag/batch_executor/mod.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod interface;
```

Comment:
It seems `interface` is not a good name in the `rust` project?

Reply:
I agree.

Reply:
Do you have some suggestions?

Reply:
Maybe we can move them into the `mod.rs`?

Reply:
I put it here so that we can `use xx::*` without worrying importing too many non-related things. If we put it in `mod` then `use xx::*` will import too many things.

Reply:
To keep consistent with the orginal `Executor`, I think put them into `mod.rs` is more reasonable?

Reply:
They are just two totally different frameworks, I don't think there are any benefits to put them together. Also, we need to wipe out the original `Executor` infrastructure totally in future, so mixing them together will just make this harder.

Reply:
I don't mean to put them together, In the PR, I think we can put the interface into `batch_executer/mod.rs`

Reply:
Got it. @rleungx It is explained in https://github.com/tikv/tikv/pull/4241#discussion_r260735869

Reply:
> I put it here so that we can `use xx::interface::*` (which is a very common case) without worrying importing too many non-related things. If we put it in `mod` then `use xx::*` will import too many things.

I'm not sure whether it is a good habit, @jay @brson what do you think?

Reply:
I don't think it's wise to spend so many time on arguing such a trivial style part, which is more like some personal tastes. We need to stay focus on principal things, i.e. logic, overall designs, considering that there are more than 3000 LOC waiting for your review in 2 weeks. If you still have concerns, I will add a TODO here so that we may consider doing something to it later.